src/DssSpell.sol

```diff
@@ -64,113 +64,126 @@ contract DssSpellAction is DssAction {
-
-        // --- GUSD PSM Parameter Changes ---
-        // Poll: https://vote.makerdao.com/polling/QmaXg3JT#vote-breakdown
-
-        // Reduce the line by 390 million DAI from 500 million DAI to 110 million DAI.
-        DssExecLib.setIlkAutoLineParameters("PSM-GUSD-A", 110 * MILLION, 50 * MILLION, 24 hours);
-        // Reduce the tout by 0.01% from 0.01% to 0%.
-        DssExecLib.setValue(MCD_PSM_GUSD_A, "tout", 0);
-
-        // Skip for Goerli
-        // --- Add Chainlink Keeper Network Treasury Address ---
-        // Forum: https://forum.makerdao.com/t/poll-notice-keeper-network-follow-up-updates/21056
-        // Forum: https://forum.makerdao.com/t/consolidated-action-items-for-2023-06-28-executive/21187
-        // Poll: https://vote.makerdao.com/polling/QmZZJcCj#vote-breakdown
-
-        // Skip for Goerli
-        // --- CU MKR Vesting Transfers ---
-        // Forum: https://mips.makerdao.com/mips/details/MIP40c3SP75#mkr-vesting
-        // ORA-001 - 297.3 MKR - 0x2d09B7b95f3F312ba6dDfB77bA6971786c5b50Cf
-
-        // Skip for Goerli
-        // --- CU MKR Vesting Transfers ---
-        // Forum: https://mips.makerdao.com/mips/details/MIP40c3SP25#mkr-vesting-schedule
-        // RISK-001 - 175 MKR - 0x5d67d5B1fC7EF4bfF31967bE2D2d7b9323c1521c
-
-        // Skip for Goerli
-        // --- CU MKR Vesting Transfers ---
-        // Forum: https://mips.makerdao.com/mips/details/MIP40c3SP17
-        // SES-001 - 10.3 MKR - 0x87AcDD9208f73bFc9207e1f6F0fDE906bcA95cc6
-
-        // Skip for Goerli
-        // --- Delegate Compensation for May (including offboarded Delegates) ---
-        // Forum: TBD
-
-        // Skip for Goerli
-        // --- BlockTower Legal Expenses DAI Transfer ---
-        // Forum: https://forum.makerdao.com/t/project-andromeda-legal-expenses/20984
-        // MIP: https://mips.makerdao.com/mips/details/MIP104#5-2-legal-recourse-asset-budget
-
-        DssExecLib.setChangelogVersion("1.14.14");
+        DssInstance memory dss = MCD.loadFromChainlog(DssExecLib.LOG);
+        FlapperInstance memory flap = FlapperInstance({
+            flapper: MCD_FLAP,
+            mom:     FLAPPER_MOM
+        });
+        FlapperUniV2Config memory cfg = FlapperUniV2Config({
+            hop:  1577 seconds,
+            want: 98 * WAD / 100,
+            pip:  PIP_MKR,
+            hump: 50 * MILLION * RAD,
+            bump: 5 * THOUSAND * RAD
+        });
+
+        FlapperInit.initFlapperUniV2({
+            dss: dss,
+            flapperInstance: flap,
+            cfg: cfg
+        });
+
+        FlapperInit.initDirectOracle({
+            flapper : MCD_FLAP
+        });
+
+        DssExecLib.setChangelogAddress("PIP_MKR", PIP_MKR);
+        // NOTE: ignore in goerli
+        // DssCronSequencerLike(CRON_SEQUENCER).addJob(CRON_FLAP_JOB);
+        // DssExecLib.setChangelogAddress("CRON_FLAP_JOB", CRON_FLAP_JOB);
+        DssExecLib.setChangelogVersion("1.15.0");
+
+        // ----- Add Cron Jobs to Chainlog -----
```

Comment:
I'd suggest leaving this before the flapper, so the sequencer and the prev cron jobs are recorded in the chainlog before the new job (that was the intention of the original order).